.github/workflows/merge-api-changelogs.yml

```diff
@@ -0,0 +1,31 @@
+name: Merge API changelogs
+
+on:
+  push:
+    branches: ["*.x", chat.zulip.org, main]
+    tags: ["*"]
+  workflow_dispatch:
```

Comment:
You should be able to filter this to only changes that touch the `unmerged.d` directories.